docs/source_files/pixi_tomls/task_minijinja_filter.toml

```diff
@@ -0,0 +1,10 @@
+
+[workspace]
+channels = ["conda-forge"]
+name = "workspace-name"
+platforms = ["linux-64"]
+# --8<-- [start:tasks]
+[tasks]
+# Convert argument to uppercase
+upper = { cmd = "echo {{ text | upper }}", args = ["text"] }
```

Comment:
Previous version of this documentation included the `join` function. Which showed me an issue with the arguments.
This is the task:
```toml
files = { cmd = "tar -czf archive.tar.gz {{ files | join(',') }}", args = ["files"] }
```

![Screenshot 2025-04-17 at 10 16 09](https://github.com/user-attachments/assets/601e71c9-da03-4459-9a19-e5f773e41be3)
 
 In this specific example, I would have expected it to accept multiple inputs for files, especially because it's only one argument. @Hofer-Julian do you remember if this was on purpose or if it is bug?